addons/event/models/event_registration.py

```diff
@@ -229,7 +229,7 @@ def action_send_badge_email(self):
             message loaded by default
         """
         self.ensure_one()
-        template = self.env.ref('event.event_registration_mail_template_badge')
+        template = self.env.ref('event.event_registration_mail_template_badge', raise_if_not_found=False)
```

Comment:
``default_template_id=template.id`` will crash as it will try to do None.id ^^

Reply:
😳 thanks! 
corrected 🙏🏻